packages/bpk-component-theme-switcher/readme.md

```diff
@@ -0,0 +1,20 @@
+# bpk-component-theme-switcher
+
+> Backpack Theme Switcher component.
+
+## Installation
+
+```sh
+npm install bpk-component-theme-switcher --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkThemeSwitcher from 'bpk-component-theme-switcher';
```

Comment:
This should have the docs for the HOC too.

Reply:
Will add, thanks

does RTL needs that as well? I copied that component and looks like it does not have it